sbin/build.sh

```diff
@@ -131,6 +131,11 @@ buildingTheRestOfTheConfigParameters()
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-cacerts-file=${WORKING_DIR}/cacerts_area/security/cacerts"
   CONFIGURE_ARGS="${CONFIGURE_ARGS} --with-alsa=${WORKING_DIR}/alsa-lib-${ALSA_LIB_VERSION}"
 
+  # Freemarker dependency?
```

Comment:
remove '?'

Reply:
i put a ? because it was only done conditionally, but I can appreciate it makes the comment sound unsure of itself :) . I'll remove it.